tools/test/perform/iopipe.c

```diff
@@ -95,7 +95,6 @@ print_stats(const char *prefix,
 static void
 synchronize(void)
 {
-#ifdef H5_HAVE_SYSTEM
 #if defined(H5_HAVE_WIN32_API) && !defined(__CYGWIN__)
     _flushall();
```

Comment:
As far as I'm aware, this only flushes FILE objects, so it's not the same as POSIX's sync(2) and is probably the wrong thing to do here. FlushFileBuffers() would be better, but that requires the file's HANDLE.